simplified-reports/src/main/java/org/nypl/simplified/reports/Reports.kt

```diff
@@ -112,7 +112,7 @@ object Reports {
           this.putExtra(Intent.EXTRA_EMAIL, arrayOf(address))
           this.putExtra(Intent.EXTRA_SUBJECT, subject)
           this.putExtra(Intent.EXTRA_TEXT, body)
-          this.putExtra(Intent.EXTRA_STREAM, arrayListOf(contentUris))
+          this.putExtra(Intent.EXTRA_STREAM, ArrayList(contentUris))
```

Comment:
Changing to `ArrayList()` instead of `arrayListOf()`. Since `contentUris` is a list, `arrayListOf(contentUris)` created a list of lists instead of a list of Uris.